examples/nodejs/FuzzTestCase.fuzz.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { FuzzedDataProvider } = require("@jazzer.js/core");
+const exploreMe = require("./ExploreMe");
+
+describe("My describe", () => {
```

Comment:
```suggestion
describe("My fuzz tests", () => {
```
or just remove it... it is not necessary